### PR TITLE
Added a checkbox in the preferences dialog for asm.indent

### LINKED ISSUE #1039: Add asm.indent in Disasm Settings

### PR DESCRIPTION
**Pull request by blu3r4d0n**
Explain the **details** for making this change. What existing problem does the pull request solve?
This allows the user to specify if they want indents to be displayed in the disasembly, it uses the asm.indent option.

**Test plan (required)**
Test to make sure this works on other platforms besides Linux.

*Unchecked Option*
unchecked
*Checked option with result* 
checked

**Closing issues**
closes #1039

### CODE REVIEW

File: src/common/Configuration.cpp [hunk 1]
```diff
@@ -24,6 +24,7 @@ static const QHash<QString, QVariant> asmOptions = {
     { "asm.pseudo",         false },
     { "asm.offset",         true },
     { "asm.xrefs",          false },
+    { "asm.indent",         false},
```
Comment (ITAYC0HEN) [inline src/common/Configuration.cpp]: formatting:
Add a space after "false"

File: src/dialogs/preferences/AsmOptionsWidget.ui [hunk 2]
```diff
@@ -52,6 +52,13 @@
          </property>
         </widget>
        </item>
+       <item>
+	<widget class="QCheckBox" name="indentCheckBox">
+	 <property name="text">
+	  <string>Show indents (asm.indent)</string>
```
Comment (ITAYC0HEN) [inline src/dialogs/preferences/AsmOptionsWidget.ui]: This isn't the description of the command. Pleas rephrase based on the output of "e? asm.indent"

File: src/common/Configuration.cpp (hunk 1 above)
Comment (philipmnelson) [reply src/common/Configuration.cpp]: Fixed in recent commit

File: src/dialogs/preferences/AsmOptionsWidget.ui (hunk 2 above)
Comment (philipmnelson) [reply src/dialogs/preferences/AsmOptionsWidget.ui]: Fixed in latest commit

**Pull request merged**